packages/core-js-sdk/test/sdk/webauthn.test.ts

```diff
@@ -173,10 +173,6 @@ describe('webauthn', () => {
 
   describe('signIn', () => {
     describe('start', () => {
-      it('should throw an error when loginId is not a string', () => {
```

Comment:
refer to the this comment about why this test was removed - https://github.com/descope/descope-js/pull/164/files#r1218491667